Example/Tests/ExternallyManagedSessionTests.swift

```diff
@@ -58,13 +58,14 @@ class ExternallyManagedSessionSpec: QuickSpec {
                         expect(spy).to(haveBeenCalled(withArgs: ["assetName": "MyAsset"]))
                     }
 
+                    // TODO: Work out why this is needed!
                     #if targetEnvironment(simulator)
                     // This test will only run on a simulator
                     // https://github.com/Quick/Nimble#swift-assertions
-                    it("throw error without source and asset name") {
-                        expect { try convivaAnalytics.initializeSession() }.to(throwError())
-                        expect(spy).toNot(haveBeenCalled())
-                    }
+//                    it("throw error without source and asset name") {
+//                        expect { try convivaAnalytics.initializeSession() }.to(throwError())
+//                        expect(spy).toNot(haveBeenCalled())
+//                    }
                     #endif
```

Comment:
IIRC the conviva session required those properties to be set at session initialization 🤔 Back then the session was shown as invalid on the validation tool (I forgot the name)

Reply:
But this is a unit test, target ONLY the simulator 🤔

Reply:
because the `throwError()` `Matcher` is not available on ARM based environments. So this does not work on M1 Macs as well. Would need a bit of love to skip them there as well.

Reply:
(Made a note of this instead of the todo 👍)